Assets/HoloToolkit/UX/Resources/Textures/console.erroricon.sml.png.meta

```diff
@@ -0,0 +1,84 @@
+fileFormatVersion: 2
+guid: e362348e990c208439be8bfb0b65b309
```

Comment:
What are these console icons for?

Reply:
The are for the debug log for different types of log files.  I'll remove them till I integrate the debug menu and debug log.

thanks,
-pat